doc/api/globals.md

```diff
@@ -972,9 +984,11 @@ added: v22.4.0
 > Stability: 1.0 - Early development.
 
 A browser-compatible implementation of [`sessionStorage`][]. Data is stored in
-memory, with a storage quota of 10 MB. Any modification of this data outside of
-the Web Storage API is not supported. Enable this API with the
-[`--experimental-webstorage`][] CLI flag.
+memory, with a storage quota of 10 MB. `sessionStorage` data persists only within
+the currently running process, and is not shared between workers.
+[`--experimental-webstorage`][] CLI flag. `sessionStorage` data is not stored per
+user or per request when used in the context of a server, it is specific to
+a single session.
```

Comment:
While it is called `sessionStorage`, there really isn't a 'session' in Node.js, so saying "it is specific to
a single session." might not make sense.

Reply:
What can we say about it ?

Reply:
I'm not sure, WDYT @cjihrig

Reply:
`['--experimental-webstorage'][] CLI flag. When using 'sessionStorage' in the context of a server, it is important to note that the data is not stored per user or per request, but rather it is specific to a single server-side context or instance.`

Maybe this ?

Reply:
@RedYetiDev @cjihrig

Reply:
```suggestion
[`--experimental-webstorage`][] CLI flag. `sessionStorage` data is not stored per
user or per request when used in the context of a server, it is specific to
a single context.
```